client/src/components/Feed/Post.js

```diff
@@ -435,7 +437,9 @@ const Post = ({
             </WebModal>
           </StyledPostPagePostCard>
           {showComments && (
-            <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
+            <StyledButtonWizard
+              nav={<WizardFormNav gtmPrefix={GTM.post.prefix} />}
+            ></StyledButtonWizard>
```

Comment:
You can change this to
```suggestion
            />
```
since there's nothing between the tags.

Reply:
Nice catch! Didn't see that